src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -102,24 +108,33 @@ public Lift() {
     }
 
     protected double returnPIDInput() {
-        return liftEncoder.getDistance();
+        return liftHeight();
     }
 
     protected void usePIDOutput(double output) {
         if (topLimitSwitch.get() == true && output > 0) {
-            setSetpoint(LiftHeight.kMaxHeight.get());
+            setSetpoint(DesiredLiftHeight.kMaxHeight.get());
             liftDrive.stopMotor();
             return;
         }
         if (bottomLimitSwitch.get() == true && output < 0) {
             liftEncoder.reset();
-            setSetpoint(LiftHeight.kFloor.get());
+            setSetpoint(DesiredLiftHeight.kFloor.get());
             liftDrive.stopMotor();
             return;
         }
         liftDrive.set(output);
     }
 
+    protected double liftLidarHeight = liftLidar.getDistance(true);
+
+    public double liftHeight() {
+        if (liftLidarHeight > 72) {
```

Comment:
Where does `72` come from?

Reply:
72 is the lidar's range in inches. after it hits its max, we will switch over to the encoders